From 9a3eb3975205c6f6cc3e2777316853575e494bbc Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 17 Jul 2020 15:03:20 +0200
Subject: [PATCH 1/2] squid: disable ASLR PIE

Because of enabled ASLR PIE by default for our builds, it does not
compile for Turris MOX. However, it passes for Turris Omnia, but it
segfaults immediately.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/squid/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/squid/Makefile b/net/squid/Makefile
index e4b261e09..3911f24c9 100644
--- a/net/squid/Makefile
+++ b/net/squid/Makefile
@@ -24,6 +24,7 @@ PKG_CPE_ID:=cpe:/a:squid-cache:squid
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
+PKG_ASLR_PIE:=0
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.27.0

